scripts/prepare-package-lock.js

```diff
@@ -3,38 +3,32 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
File is a copy of: 
https://github.com/cloudscape-design/components/blob/main/scripts/unlock-package-lock.js